src/components/Data/Collections/Dropdown.vue

```diff
@@ -1,68 +1,88 @@
 <template>
   <span>
-    <dropdown :id="'collection-' + collection" :myclass="myclass">
-      <li>
-        <router-link
-          :class="{ disabled: !canEditCollection(index, collection) }"
-          :to="
-            canEditCollection(index, collection)
-              ? { name: 'DataCollectionEdit', params: { collection, index } }
-              : ''
-          "
-          >Edit collection</router-link
-        >
-      </li>
-      <li v-if="isRealtime">
-        <a class="remove" @click="removeRealtimeCollection"
-          >Remove collection</a
-        >
-      </li>
-      <li class="divider" />
-      <li v-if="!isRealtime && !isList">
-        <router-link
+    <b-dropdown
+      data-cy="CollectionDropdown"
+      no-caret
+      toggle-class="collectionDropdown"
+      variant="light"
+      :id="'collection-' + collection"
```

Comment:
not possible to use es6 Template literals `collection-$[collection}` here ?

Reply:
yes of course